test/src/tests-main.js

```diff
@@ -29,31 +33,31 @@ afterEach(function() {
 });
 
 import './tests-core-sdk';
-import './tests-temp-session-bug-fix';
-import './tests-batchUploader';
-import './tests-beaconUpload';
-import './tests-kit-blocking';
-import './tests-migrations';
-import './tests-persistence';
-import './tests-forwarders';
-import './tests-helpers';
-import './tests-identity';
-import './tests-event-logging';
-import './tests-eCommerce';
-import './tests-cookie-syncing';
-import './tests-identities-attributes';
-import './tests-native-sdk';
-import './tests-consent';
-import './tests-serverModel';
-import './tests-mockBatchCreator.ts';
-import './tests-mParticleUser';
-import './tests-self-hosting-specific';
-import './tests-runtimeToBatchEventsDTO';
-import './tests-apiClient';
-import './tests-mparticle-instance-manager';
-import './tests-queue-public-methods';
-import './tests-validators';
-import './tests-utils';
-import './tests-store';
-import './tests-config-api-client';
-import './tests-vault';
+// import './tests-temp-session-bug-fix';
```

Comment:
Nitpick. I think I understand at this moment why you're commenting this block of code out, but we should write a comment explaining why, since Future Alex and Rob are probably exhausted and braindead. Especially Future Alex.

Reply:
i'll add it in the next one.   i expect this to move along pretty quickly at which point there won't be anything commented anymore.